WalletWasabi.Fluent/Views/Wallets/MusicControlsView.axaml

```diff
@@ -143,15 +143,19 @@
                         <PathIcon Data="{StaticResource coordinator}" />
                       </MenuItem.Icon>
                     </MenuItem>
-                    <MenuItem Header="How do I setup a coordinator?" Command="{Binding CoordinatorHelpCommand}" />
+                    <MenuItem Header="How do I setup a coordinator?" Command="{Binding CoordinatorHelpCommand}" ToolTip.Tip="{Binding FindCoordinatorLink}" >
```

Comment:
```suggestion
                    <MenuItem Header="How do I find & set up a coordinator?" Command="{Binding CoordinatorHelpCommand}" ToolTip.Tip="{Binding FindCoordinatorLink}" >
```

Reply:
Funnily enough this commit broke the build, & is not a valid char in an avalonia axaml as it seems. Probably can be escaped, I removed it